dialog.txt

```diff
@@ -36,70 +36,70 @@ Errore di sistema!!!#|
-*Wow! That Canary# is the baby of# singing Laura.#_*Will you return her# baby to her.#|
-*Wait...#_*You just missed# something very# important.#|
-*Where is Laura?...#|
-*Laura remembered# her song, and began# to sing once more.#_*Did you learn# the melody?#|
-*That's no good.#|
-*The truth is...# There is power in# song.#|
-*Hi! I am Pippi,# and I'm lost.#_*What a brave# boy you are to# have found me.#|
-*Don't get so# swell headed.# Ha-ha-ha.#|
-*You're brave. I# hope we can meet# again sometime.#|
-*I'm so happy!  To# show my gratitude# take my gift.#_*This shiny Badge# will guard you.# Keep it!#|
-*You probably# already have# someone special,#|
-*Now, it's time to# say good-bye.#_*Please come by my# house, I'll be# waiting there to#_*see you again.# By the way...#_*Do you think my# hair needs# styling?#|
-*You didn't have to# answer, cutie.#|
+*I pulcini al# Villaggio Canarino# hanno delle voci# incantevoli.#|
+*Il signore del# Villaggio Canarino# è il mio idolo.#|
+*Il Sindaco sta# cercando un uomo# audace e impavido.#|
+*La bambina che# si è smarrita nel#_*cimitero è # stata salvata?#|
+*Stai...# S-stai veramente# andando al cimitero?#|
+*Pensavo che fossi# un bimbo ragionevole.#|
+*Gli animali sembrano# impazziti, sono# fuori controllo.#_*I morti sono diventati# zombi e stanno# attaccando la gente.#_*Che possiamo fare?#|
+*Sei magnifico!#|
+*Per caso uno spettro# è venuto a casa tua?#_*A me l'ha distrutta.#|
+*Gli animali sono# impazziti e stanno# scappando,#_*per cui lo Zoo# è stato chiuso.#|
+*C'è del pane raffermo# se vuoi MANGIAre,#_*ma usando le sue# Briciole puoi segnare# il tuo sentiero.#_*Dici che sto# parlando troppo?# E allora?#|
+*Cittadino, so# che vorresti soltanto# aiutare la città.#|
+*Dunque...# Un bimbo si è# perso nel cimitero.#_*Ed io...# candidando per le# prossime elezioni.#_*Devi aiutarmi!# Ti prego!#|
+*Uh!?# Andrai lì# per salvarla?#|
+*Ma lei è la ragazza# perduta, Pippi?#_*Beh ma è# favoloso.#_*Sapevo che avresti# compiuto il tuo# dovere da cittadino.#_*Applausi,# applausi...#_*Ora piccola Pippi,# non dimenticare...#_*di dire a mamma# che il Sindaco# ti ha salvata.#_*Tieni, ti do# ¹£@ un premio di# ³ª|§|.#|
+*A proposito, eroe,#_*forse ti piacerebbe# andare a vedere# che succede allo Zoo?#|
+*Sì, ci andrai?# Fantastico, fantastico# fantastico!#_*Assicurati di prendere# una chiave dal mio# assistente, Abbott.#|
+*Questa chiave aprirà il# cancello dello zoo.# Non perderla.#|
+*Restituiscimela viva,# e sicuramente# sarò un eroe...#_*Ehm, umh, voglio dire# sarai un eroe.#|
+*Perché se non è il# nostro eroe ¹£@,#_*Hai dello sterco di# tigre sui tuoi# vestiti?#_*Ha-ha, sto solo# scherzando. Adesso vai# ragazzo.#|
+*Ho sentito delle# voci su una# ragazza prodigiosa.#_*Ti aiuterà# nel tuo viaggio.#|
+*Le chiavi dello Zoo# te le tirano dietro,# prendine un'altra.#|
+*Dai un'occhiata,# tutti gli animali# sono scappati.#|
+*È rimasto solo questo# pulcino di canarino.# Vuoi comprarlo?#|

```

Comment:
Cambierei in "Pulcino di Canarino", visto che i nomi degli oggetti hanno le maiuscole.